moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummary.js

```diff
@@ -60,6 +60,12 @@ const useStyles = makeStyles((theme) => ({
     fontSize: ".8rem",
     paddingLeft: theme.spacing(0.5),
   },
+  fieldLabelSmall: {
+    width: "100%",
+    color: theme.palette.text.secondary,
+    fontSize: ".7rem",
```

Comment:
the issue says to make smaller, the existing size is .8rem. I tried .6rem and it looked a bit too small.

Reply:
i do me; you do you